### PR TITLE
Automatic port of Microsoft.Extensions.Logging.AzureAppServices

### PR DESCRIPTION
**Pull request by carlossanlop**
Area owners of Microsoft.Extensions.Logging: @maryamariyan @tarekgh

### CODE REVIEW

**Comment by maryamariyan** [comment]
Microsoft.Extensions.Logging.AzureAppServices moved to dotnet/aspnetcore (Refer to [this](https://github.com/aspnet/Announcements/issues/411)). Perhaps their docs don't belong to dotnet/dotnet-docs-api.

@pranavkm @ericstj thoughts?

**Comment by ericstj** [comment]
I don't really have a preference.  There are plenty of libraries documented here which don't build out of dotnet/runtime nor dotnet/extensions.  @pranavkm what is your preference?  Do you want all ASPNet docs in a different repo?

File: xml/Microsoft.Extensions.Logging.AzureAppServices.Internal/WebAppContext.xml [hunk 1]
```diff
@@ -99,7 +99,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets a value indicating whether or new we're in an Azure WebApp.</summary>
```
Comment (maryamariyan) [inline xml/Microsoft.Extensions.Logging.AzureAppServices.Internal/WebAppContext.xml:102]:
```suggestion
        <summary>Gets a value indicating whether or not we're in an Azure WebApp.</summary>
```

**Review by maryamariyan — APPROVED** [review]

**Comment by carlossanlop** [comment]
Warning is happening also in a line unrelated to this PR, @TianqiZhang @joelmartinez :

File | Status | Preview URL | Details
-- | -- | -- | --
xml/Microsoft.Extensions.Logging.AzureAppServices/BatchingLoggerProvider.xml | Warning | View | Line 0: [Warning] Cross reference not found: 'Microsoft.Extensions.Logging.AzureAppServices.BatchingLogger'.Line 0: [Warning] Cross reference not found: 'Microsoft.Extensions.Logging.AzureAppServices.BatchingLogger'.Line 0: [Warning] Cross reference not found: 'Microsoft.Extensions.Logging.AzureAppServices.BatchingLogger'.

**Comment by joelmartinez** [comment]
@carlossanlop hmm ... the actual type seems to be in a different namespace than the one mentioned in those warnings:  
`Microsoft.Extensions.Logging.AzureAppServices.BatchingLogger`

https://github.com/dotnet/dotnet-api-docs/blob/6a4a5579c7e5e9161443959ff4b33a2b0b0d9199/xml/Microsoft.Extensions.Logging.AzureAppServices.Internal/BatchingLogger.xml
`T:Microsoft.Extensions.Logging.AzureAppServices.Internal.BatchingLogger`

**Pull request merged**